runners/flink/runner/pom.xml

```diff
@@ -101,6 +101,17 @@
 
     <dependency>
       <groupId>org.apache.beam</groupId>
+      <artifactId>runners-core</artifactId>
```

Comment:
Ah, I was talking about @davorbonaci's comment about adding a Spark dep and a data flow dep.

The intention of the other stuff was pretty clear. 😃